regolith/helpers/u_finishprumhelper.py

```diff
@@ -18,6 +18,10 @@ def subparser(subpi):
     subpi.add_argument("-d", "--database",
                        help="The database that will be updated.  Defaults to "
                             "first database in the regolithrc.json file.")
+    subpi.add_argument("--pub_sum", type = str,
```

Comment:
please change help message to
"The public summary. This will be used in, for example, as web news items. It should capture in a single paragraph the  main context, result and importance without jargon in words that a member of the general public can understand.  This is required to finish a prum"